live-examples/js-examples/string/string-length.html

```diff
@@ -0,0 +1,5 @@
+<pre>
+<code id="static-js">const str='Life, the universe and everything. Answer:';
```

Comment:
Stick with ES5 for ES5 examples (https://github.com/mdn/interactive-examples/blob/master/JS-Example-Style-Guide.md#language-choice-es5es6) - this means using `var` here.

Reply:
Also please use spaces for readability: `var str = '...`